tests/lib/rules/no-debug.test.ts

```diff
@@ -87,6 +87,19 @@ ruleTester.run(RULE_NAME, rule, {
         screen.debug
       `,
     },
+    {
+      code: `
+        import { screen } from '@testing-library/dom'
+        screen.logTestingPlaygroundURL()
+      `,
+    },
+    {
+      code: `
+        import { screen } from '@testing-library/dom'
+        screen.debug()
+      `,
+      options: [{ utilNames: ['anotherUtil'] }],
```

Comment:
This test should not be correct after you set the option type properly, so it will have to be updated.